packages/node_modules/@ciscospark/widget-space/src/container.js

```diff
@@ -417,18 +363,7 @@ export class SpaceWidget extends Component {
       const callStartTime = media.get(`callStartTime`);
       return (
         <div className={classNames(`ciscospark-space-widget`, styles.spaceWidget)}>
-          {
-            errors.get(`hasError`) &&
-            <div className={classNames(`ciscospark-error-wrapper`, styles.errorWrapper)}>
-              <ErrorDisplay
-                actionTitle={errorActionTitle}
-                onAction={errorAction}
-                secondaryTitle={displaySubtitle}
-                title={displayTitle}
-                transparent={temporary}
-              />
-            </div>
-          }
+          { errors.get(`hasError`) && errorElement }
```

Comment:
since we aren't populating `errorElement` above unless `hasError` you could probably just put `{ errorElement }` in

Reply:
Good call